users/validation.py

```diff
@@ -0,0 +1,21 @@
+import re
+
+from django.forms import ValidationError
```

Comment:
저희는 form을 사용하지 않기때문에 django.core.exceptions 로부터 import 하는 방법으로 변경해주세요!
```suggestion
from django.core.exceptions import ValidationError
```

Reply:
네! 수정하겠습니다!